client/nginx.go

```diff
@@ -193,6 +193,20 @@ type NginxInfo struct {
 	ParentProcessID uint64 `json:"ppid"`
 }
 
+// LicenseReporting contains information about license status for NGINX Plus.
+type LicenseReporting struct {
+	Healthy bool
+	Fails   uint64
+	Grace   uint64
+}
+
+// NginxLicense contains licensing information about NGINX Plus.
+type NginxLicense struct {
+	ActiveTill uint64 `json:"active_till"`
+	Eval       bool
```

Comment:
Do we need json comments on these also, and the struct above, to tell it its lowercase?

Reply:
No, you only need the json hints if the parameters don't match, the capital in the struct doesn't matter